packages/solid-crs-manage/lib/features/object/object-root.component.ts

```diff
@@ -160,11 +170,37 @@ export class ObjectRootComponent extends RxLitElement {
       ${ this.state?.matches(ObjectStates.EDITING) ? html`<div slot="actions"><button class="no-padding inverse cancel" @click="${() => this.actor.send(ObjectEvents.CANCELLED_EDIT)}">${unsafeSVG(Cross)}</button></div>` : '' }
       <div slot="actions"><button class="no-padding inverse delete" @click="${() => this.actor.send(ObjectEvents.CLICKED_DELETE, { object: this.object })}">${unsafeSVG(Trash)}</button></div>
     </nde-content-header>
+    <div class="content-and-sidebar">
+
+    <nde-sidebar>
+      <nde-sidebar-item>
+        <div slot="content">
+          Een week geleden aangepast door Pieter Pieters
+        </div>
+      </nde-sidebar-item>
```

Comment:
I'd leave this out for now, as we can't fill it in with anything useful. I think @wouteraj said the same last week.